server/api.go

```diff
@@ -22,6 +22,12 @@ const (
 	API_ERROR_ID_NOT_CONNECTED = "not_connected"
 )
 
+type OauthState struct {
```

Comment:
```suggestion
type OAuthState struct {
```
If you accept, will need to update the other uses of it as well

Reply:
Changed, thanks!